webui/package-lock.json

```diff
@@ -1,8 +1,7715 @@
 {
   "name": "lakefs-ui",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This happens when we use npm v7 or above.  It is probably before you are using newer version of nodejs.
We can upgrade the current node version requirements, update the contribution doc and workflows - but maybe in a different pr.

Reply:
I'm using npm 8.5.2, and node v14.18.1 (which is what our contribution guide has). 
Does this make sense that I'm getting this change?

Reply:
https://nodejs.org/en/download/releases/
There is the node+npm version out of the box and the latest.
my concern was that we ask for node 14 that comes with an older npm, and we can use a newer node.